global.css

```
@@ -38,3 +38,9 @@ body {
   margin: 0;
   font-size: 70px;
 }
+.about {
+  font-family: minecraft;
+  margin: 0;
+  font-size: 70px;
+  align-items: center;
```

```suggestion
  text-align center;
```

If you **do want all of the items in the about section with the same font size and text centred**, you need to use `text-align` to dicate where the text is. `align-items` can only be used if the parent HTML tag has a display of flex.

---

But if i want only the title "What is minecraft?" in the "MinecraftTen-VGORe.ttf" style.